contracts/utility/TokenForwarding.cdc

```diff
@@ -16,27 +16,27 @@ their tokens to.
 
 import FungibleToken from "FungibleToken"
 
-pub contract TokenForwarding {
+access(all) contract TokenForwarding {
 
     // Event that is emitted when tokens are deposited to the target receiver
-    pub event ForwardedDeposit(amount: UFix64, from: Address?)
+    access(all) event ForwardedDeposit(amount: UFix64, from: Address?)
```

Comment:
It would be nice if this event had type and uuid fields like new FT/NFT events has. Atm if i have a tx that has 10 flow and 10 usdc and forwards both it is hard to know what goes where just looking at this event. 

Come to think of it, how will the forwarders work with the standard Deposit/Withdraw events?

Reply:
I'll update this. The forwarders work the same way. standard Deposit events only get emitted by collections, so in a forwarded deposit, you would see one forwarded deposit event and then one regular deposit event